### DISCUSSION TITLE
Does Vercel support Docker if it's wrapped up with Turborepo?

### DISCUSSION THREAD

**Discussion by corysimmons** [vercel/turborepo#1708] [Help] [answered]
- Turborepo supports Docker
- Vercel doesn't support Docker
- So Vercel doesn't support Turborepo deployments (if they have Docker in them)?

Is this correct?

**Comment by EarlGeorge** [comment, marked as answer]
**Vercel** doesn't support docker deployment, but u can use **Turborepo** and docker to deploy it elsewhere

**Reply by corysimmons** [reply]
Thank you. Do you happen to know of any other hosts that are super friendly with Turborepo? Specifically, I love how Turborepo leans into monorepos and my builds are fast as heck, but I need to use Docker (for Hasura).

**Reply by EarlGeorge** [reply]
if you dockerize your app efficiently u can host it even on **Mars**

**Reply by corysimmons** [reply]
mars.com is something about global pet care and mars.sh is down. 😩 Guess it's finally time I learn AWS good and proper.

**Reply by EarlGeorge** [reply]
I meant Planet, it was funny 😄 nah, you don't need AWS maybe only couple services, but give it a try if you have some free time.
